app/components/items/clover/audio/viewer.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
```

Comment:
I haven't checked the clover-iiif docs but this has to be a client component, right?

Reply:
i didn't see it mentioned in the clover-iiif docs, but I ran into issues trying to render it in the app router without "use client"